test/clj/cider/nrepl/middleware/test_test.clj

```diff
@@ -152,7 +152,11 @@
     (is (= "custom-output\n"
            (#'test/print-object (with-meta {:not :printed} {:type ::custom}))))
     (is (= "{:a :b, :c :d}\n"
```

Comment:
This is a somewhat deceitful example, because a map of symbols printed by `println` and `pprint` looks identical:

```
user=> (println {:a :b :c :d})
{:a :b, :c :d}
nil
user=> (clojure.pprint/pprint {:a :b :c :d})
{:a :b, :c :d}
nil
```

the differences become apparent once you add strings, symbols, and other data:

```
user=> (println {:a "string" :c "42" :d 'symbol})
{:a string, :c 42, :d symbol}
nil
user=> (clojure.pprint/pprint {:a "string" :c "42" :d 'symbol})
{:a "string", :c "42", :d symbol}
nil
```

Reply:
Nice observation! These tests are good to keep, however please write them in the form `(is (= ...) "pprint is chosen, as it can be seen in the quotation marks")`